app/graphql/types/query_type.rb

```diff
@@ -5,6 +5,14 @@ class QueryType < Types::BaseObject
     field :get_user, Types::UserType, null: false, description: 'Returns a single user by id' do
       argument :id, ID, required: true
     end
+
+    # Events
+    field :get_events, [Types::EventType], null: false, description: 'Returns a list of events'
```

Comment:
We will be deleting this one, right? @dhalverson

Reply:
Yes! I have deleted this dang line like 4 times now. Updated

Thank you!